app/controllers/news_letter_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class NewsLetterController < ApplicationController
+  def index
+    validate_news_letter_email_is_valid
+    GoogleSpreadSheet.new.create_news_letter_email_in_google_sheets(params)
+    respond_to do |format|
+      format.html 
+      format.js 
+    end
+    flash[:notice] = "Thanks for subscribing! You will receive an e-mail confirmation shortly."
+  rescue StandardError => e
+    flash[:error] = e.message
```

Comment:
Would you mind logging to ERROR level on this?  Or would INFO be more appropriate here?

Reply:
Error is good.